app/views/layouts/application.html.erb

```diff
@@ -3,7 +3,6 @@
   <head>
     <title>Quibble</title>
     <%= stylesheet_link_tag    "application", media: "all" %>
-    <%= stylesheet_link_tag    "materialize", media: "screen, projection" %>
```

Comment:
It's not a big deal if we don't care about making the website responsive and don't want to use any icons, but it would be nice to have both of those options stay open if possible.